sormas-backend/pom.xml

```diff
@@ -39,6 +39,11 @@
 			<artifactId>commons-validator</artifactId>
 		</dependency>
 
+		<dependency>
```

Comment:
Why this dependency import?

Reply:
Forgot to remove it after settling with the right exception class